pkgs/development/tools/misc/pkg-config/default.nix

```
@@ -35,8 +35,12 @@ stdenv.mkDerivation rec {
          "ac_cv_func_posix_getgrgid_r=yes"
        ];
 
-  # Silence "incompatible integer to pointer conversion passing 'gsize'" when building with Clang.
-  env.NIX_CFLAGS_COMPILE = lib.optionalString stdenv.cc.isClang "-Wno-int-conversion";
+  env.NIX_CFLAGS_COMPILE = builtins.toString (
+    # Silence "incompatible integer to pointer conversion passing 'gsize'" when building with Clang.
+    lib.optionals stdenv.cc.isClang ["-Wno-int-conversion"] ++
+    # Silence fprintf format errors when building for Windows.
+    lib.optionals stdenv.hostPlatform.isWindows ["-Wno-error=format"]
```

```suggestion
    lib.optionals stdenv.cc.isClang ["-Wno-int-conversion"]
    # Silence fprintf format errors when building for Windows.
    ++ lib.optionals stdenv.hostPlatform.isWindows ["-Wno-error=format"]
```

Make it a bit more consistent with the code style of this file (optional).